### DISCUSSION TITLE
REST API?

### DISCUSSION THREAD

**Discussion by ShadiestGoat** [plan-player-analytics/Plan#2408] [APIs and Development]
Is there a rest api for this bad boy? 

### Additional context
I want to use a bot & on this, and it isnt exactly the cleanest method of work to put my hands deep inside a database and fetch the values out. I am looking through the wiki, but perhaps I am just blind?

**Comment by AuroraLS3** [comment]
It's hidden https://github.com/plan-player-analytics/Plan/blob/master/Plan/json-endpoints.md

**Comment by ShadiestGoat** [comment]
Ah thank you for clarifying ^^
